content/scout/policy/index.md

```diff
@@ -0,0 +1,172 @@
+
+This policy is unfulfilled if an artifact is affected by one or more
+vulnerability where a fix version is available.
+
+### Critical vulnerabilities
+
+This policy requires that your artifacts contain no known critical
+vulnerabilities. The policy is unfulfilled if your artifact contains one or
+more critical vulnerabilities.
+
+This policy flags all critical vulnerabilities, whether or not there's a fix
+version available.
+
+### Packages with GPL3+ licenses
+
+This policy requires that your artifacts don't contain packages distributed
+under a GPL3+ [copyleft](https://en.wikipedia.org/wiki/Copyleft) license.
+
+This policy is unfulfilled if your artifacts contain one or more packages with
+a violating license.
+
+## View policy status
+
+The **Overview** tab of the [Docker Scout Dashboard](https://scout.docker.com/)
+displays a summary of recent changes in policy for your repositories.
+This summary shows images that have seen the most change in their policy
+evaluation between the most recently analyzed tag and the previous tag.
+
+![Policy overview](../images/policy-overview.png)
+
+### Policy status per repository
+
+The **Images** tab shows the current policy status, and recent policy trend,
+for all images in the selected environment. The **Policy status** column in the
+list shows:
+
+- Number of fulfilled policies versus the total number of policies
+- Recent policy trends
+
+![Policy status in the image list](../images/policy-image-list.png)
+
+The policy trend, denoted by the directional arrows, indicates whether a tag is
+better, worse, or unchanged in terms of policy, compared to the previous tag in
+the same environment.
+
+- The green arrow pointing upwards shows the number of policies that got better
+  in the latest version of this image.
+- The red arrow pointing downards shows the number of policies that got worse
+  in the latest version of this image.
+- The gray arrow pointing sideways shows the number of policies that were
+  unchanged in the latest version of this image.
+
+If you select a repository, you can open the **Policy** tab for a detailed
+description for the policy delta for the most recently analyzed tag and its
+predecessor.
+
+### Detailed evaluation results
+
+To view the full results of a policy evaluation for a tag, select the tag in
+the Docker Scout Dashboard and open the **Policy** tab. This shows a breakdown
+for all policy violations for the current tag.
+
+![Detailed tag evaluation results](../images/policy-detailed-results.png)
+
+### Policy change by external factors
+
+Sometimes, the **Policy status** column in the **Images** tab displays an
+additional warning or informational icon.
+
+![Icons next to policy status](../images/policy-status-icons.png)
+
+Policy evaluation results are not static. Policy status for a tag can change if
+new information becomes available, or if your policy configuration changes.
+These icons appear when the policy status for a tag changed due to external
+factors.
```

Comment:
**issue:** Based on [a discussion](https://docker.slack.com/archives/C04HJ2TKLUV/p1692705236913149?thread_ts=1692701138.379119&cid=C04HJ2TKLUV) we have removed the external factors icon so users wouldn't see it. We've also switched to showing just the previous image delta so we will not surface any delta caused by external factors.

I believe this paragraph should be omitted for now.

Reply:
@Daniel-Kolev does [this update](https://docker.slack.com/archives/C04HJ2TKLUV/p1692809250138139?thread_ts=1692807963.923709&cid=C04HJ2TKLUV) from Kipz impact your suggestion here?

Reply:
Nope. My proposal still holds